src/style/sections/management-item.css

```diff
@@ -0,0 +1,107 @@
+.management-item {
+  padding: 60px 20px;
+}
+
+.heading-content {
+  flex-direction: column;
+  text-align: center;
+}
+
+.management-item-heading {
+  font-family: ManropeExtraBold, sans-serif;
+  font-size: var(--font-size-md);
+  font-weight: var(--font-weight-extra-bold);
+  line-height: var(--line-height-4sm);
+  color: var(--color-black);
+}
+
+.management-item-decs {
+  margin-top: 15px;
+  font-size: var(--font-size-4xs);
+  font-weight: var(--font-weight-default);
+  line-height: var(--line-height-3xs);
+  color: var(--color-black);
+  letter-spacing: 0.2px;
+}
+
+.management-item-category-list {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 80px;
+  margin-top: 40px;
+}
+
+.management-item-category:nth-child(2) .circle-icon {
```

Comment:
Is it still right when the item change the order in different devices (laptop, tablet)?